tests/test-sources/plugins/git/octo.nix

```diff
@@ -0,0 +1,68 @@
+{
+  empty = {
+    plugins.octo.enable = true;
+  };
+
+  example = {
+    plugins.octo = {
+      enable = true;
+
+      settings = {
+        ssh_aliases = {
+          "github.com-work" = "github.com";
+        };
+        mappings_disable_default = true;
+        mappings = {
+          issue.react_heart = "<leader>rh";
+          file_panel.select_prev_entry = "[q";
+        };
+        picker = "telescope";
+      };
+    };
+  };
```

Comment:
```suggestion
      };
    };
  };
  
  withPicker = {
    plugins = {
      telescope.enable = true;
      
      octo = {
        enable = true;
        settings.picker = "telescope";
    };
  };
```

I think _something_ along these lines would fix CI

Reply:
ci issues were fixed by rebasing main. there's no need to add this test